raiden/tests/scenarios/pfs7_multiple_payments.yaml

```diff
@@ -0,0 +1,90 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: any
+  services:
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+        balance_per_node: 20000
```

Comment:
Please also set the "max_funding" parameter, since that needs to be >= than balance_per_node.
`max_funding: 20000`

This is related to Issue #249